integration/smoke/run_smoke.sh

```diff
@@ -58,8 +58,10 @@ function run_all_power_sweep {
     APPLICATIONS="dgemm dgemm_tiny nekbone minife amg nasft hpcg"
     EXP_SUBDIR=power_sweep
     EXP_TYPE=power_sweep
-    ARGS="--min-power=220 --max-power=230"
+    ARGS="--min-power=190 --max-power=230"
     run_all
+    python3 ${EXP_DIR}/power_sweep/gen_policy_recommendation.py \
```

Comment:
It would be better to put this in the gen_smoke script if possible.

Reply:
Did I do this right?

Reply:
yes, it's in the right file now, thanks.